src/plugins/Dashboard/index.js

```diff
@@ -201,6 +201,20 @@ module.exports = class Dashboard extends Plugin {
     }
   }
 
+  scrollBehaviour (toggle) {
+    if (!this.opts.disablePageScrollWhenModalOpen) return
+    const body = document.querySelector('body')
+    switch (toggle) {
+      case 'enable':
+        Object.assign(body.style, { overflow: 'initial', height: 'initial' })
+        break
+      case 'disable':
+        Object.assign(body.style, { overflow: 'hidden', height: '100vh' })
+        break
```

Comment:
this looks a bit odd to me, could we just have two separate methods, eg `enableScrollBehavior` and `disableScrollBehavior`, instead?